recipes/libssh/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import collect_libs, copy, get, rmdir
+
```

Comment:
```suggestion

required_conan_version = ">=1.47.0"

```
Check which conan version is required.